### ISSUE TITLE
Mismatch between reported and observed performance of pretrained models made available

### ISSUE THREAD

**Issue by Aradhye2002** [wl-zhao/VPD#31]
I am getting the following metrics on the 654 test set of images with the vpd_depth_480x480.pth file (obtained from the hyperlink "Tsinghua Cloud" in depth.md) as the checkpoint:

<img width="819" alt="Screenshot 2023-08-13 at 1 46 59 PM" src="https://github.com/wl-zhao/VPD/assets/20011003/f1c8093a-bde7-49b9-9ece-fcf8663bbbe3">

The reported metrics are:

<img width="679" alt="Screenshot 2023-08-13 at 1 47 46 PM" src="https://github.com/wl-zhao/VPD/assets/20011003/84467df0-fa9b-4712-bd93-99858ef0e041">

Why am I observing this degradation in performance for the same model? Am I doing something wrong?

**Comment by cdforigua05** [comment]
Hi @Aradhye2002 , 
Could you reproduce the results in the test set? I'm trying to reproduce the results of the paper but I'm getting higher numbers. Did you found anything that could explain the mismatch?
Thanks

**Comment by Aradhye2002** [comment]
Hi David,

Yes I was able to reproduce the results.

Actually earlier I had been experimenting with the code to understand its
working. I must have done some change (such as zeroing out the text
embeddings) to see the effect which might have resulted in degradation.

I suspected this was the case and decided to do a fresh git clone of the
repo. Now on evaluation I obtained the same results.

Hope it helps.
